helm/hpcc/values.schema.json

```diff
@@ -1171,6 +1171,25 @@
           "type": "boolean",
           "description": "If true, sets OTEL library logging to debug level (otherwise set to warning)"
         },
+        "sampler": {
```

Comment:
question: Would "sampling" be more intuitive to the user?

Reply:
I think the usability improvement would be negligible.